cmd/securityContext.go

```diff
@@ -12,39 +12,37 @@ func checkSecurityContext(container apiv1.Container, result *Result) {
 	result.capsDropped = true
 
 	if container.SecurityContext == nil {
-		result.err = 1
+		result.err = ErrorSecurityContextNIL
 		return
 	}
 
 	if container.SecurityContext.Capabilities == nil {
-		result.err = 2
+		result.err = ErrorCapabilitiesNIL
 		return
 	}
 
 	if container.SecurityContext.Capabilities.Add != nil {
-		result.err = 3
+		result.err = ErrorCapabilitiesAddedOrNotDropped
 		result.capsAdded = container.SecurityContext.Capabilities.Add
 	}
 
 	if container.SecurityContext.Capabilities.Drop == nil {
-		result.err = 3
+		result.err = ErrorCapabilitiesAddedOrNotDropped
 		result.capsDropped = false
 	}
-
-	return
 }
 
 func printResultSC(results []Result) {
 	for _, result := range results {
 		switch err := result.err; err {
-		case 1:
+		case ErrorSecurityContextNIL:
```

Comment:
Can you standardize the logging to be in line with what we do in here: https://github.com/Shopify/kubeaudit/blob/master/cmd/image.go#L33